static/compiler.js

```diff
@@ -678,7 +678,10 @@ define(function (require) {
 
     Compiler.prototype.onCfgViewClosed = function (id, network) {
         if (this.id == id) {
-            this.cfgButton.prop('disabled', false);
+            var state = false;
+            if (this.filters.binary) 
+                state = true;
+            this.cfgButton.prop('disabled', state);
```

Comment:
This can all be simplified to `this.cfgButton.prop('disabled', this.filters.binary);`

Reply:
pfffff! I hate it when I don't see obvious things like that!